src/listener_message.rs

```diff
@@ -15,18 +15,11 @@
 // Please review the Licences for the specific language governing permissions and limitations
 // relating to use of the SAFE Network Software.
 
-use sodiumoxide::crypto::sign::PublicKey;
 use socket_addr::SocketAddr;
 
+pub const REQUEST_MAGIC_CONSTANT: [u8; 4] = ['E' as u8, 'C' as u8, 'H' as u8, 'O' as u8];
```

Comment:
I think so, or if we say this connection can only ever get 1 message type then we can almost ignore the enum. It's more where we are trying to identify/dispatch message types we could receive an enum or identifier is required. IF there is a sinlge type then we can ignore this at the moment I think.